pkg/client/events/replay_client.go

```diff
@@ -27,19 +27,47 @@ const (
 	// eventsBytesRetryDelay is the delay between retry attempts when the events
 	// bytes observable returns an error.
 	eventsBytesRetryDelay = time.Second
+)
+
+// ConnStateStatus represents the status of the underlying websocket connection
```

Comment:
Is this JUST for websockets or any connection?

Should we rename it and/or move it to a "websocket appropriate" file?